src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -57,6 +57,19 @@
         <!-- CoreExtension -->
         <service id="form.property_accessor" alias="property_accessor" public="false" />
 
+        <service id="form.choice_list_factory.default" class="Symfony\Component\Form\ChoiceList\Factory\DefaultChoiceListFactory" public="false"/>
+
+        <service id="form.choice_list_factory.property_access" class="Symfony\Component\Form\ChoiceList\Factory\PropertyAccessDecorator" public="false">
+            <argument type="service" id="form.choice_list_factory.default"/>
+            <argument type="service" id="property_accessor"/>
```

Comment:
>So I guess the point is that people can override them with different ones?

It is. For example I had to override the property accessor used by the serializer to have a more perform one for my app: the Symfony one relies on reflection and allow a bunch of conventions.